recipe/meta.yaml

```
@@ -23,42 +20,42 @@ requirements:
     - setuptools
     - pip
     - wheel
+    - flit-core <4
 
   run:
     - python
-    - nltk >=3.1
+    - nltk >=3.8
 
-{% set tests_to_ignore = "--ignore=tests/tests/test_blob.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_classifiers.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_np_extractor.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_tokenizers.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_sentiments.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_taggers.py" %}
 
 test:
   source_files:
     - tests
+    - pyproject.toml
   imports:
     - textblob
     - textblob.en
     - textblob.unicodecsv
   commands:
     - pip check
-    - pytest -v tests {{ tests_to_ignore }}
+    # Downloading nltk assets for testing. textblob.download corpora is insufficient for testing needs.
+    - python -c "import nltk; nltk.download('punkt_tab')"
+    - python -c "import nltk; nltk.download('averaged_perceptron_tagger_eng')"
+    - python -m textblob.download_corpora
+    - pytest -v tests
   requires:
     - pip
-    - nose >=1.3.0
-    - mock >=3.0.5
-    - invoke >=0.15.0
     - pytest
+    # Adding numpy to test some of the classifying features of the library
+    - numpy
+
 
 about:
   home: https://github.com/sloria/TextBlob
   license: MIT
   license_file: LICENSE
```

```suggestion
  license: MIT AND BSD-2-Clause AND BSD-3-Clause
  license_file:
    - LICENSE
    - NOTICE
```
The project itself is MIT, but it ships with code that is both BSD-2 and BSD-3 as explained in the [NOTICE](https://github.com/sloria/TextBlob/blob/0.18.0.post0/NOTICE) file.
* https://github.com/sloria/TextBlob/blob/0.18.0.post0/src/textblob/_text.py is BSD-3-CLAUSE
* https://github.com/sloria/TextBlob/blob/0.18.0.post0/src/textblob/unicodecsv/__init__.py is BSD-2-CLAUSE

They also say that they bundle nltk, but that's not true anymore since 0.9.0 (https://github.com/sloria/TextBlob/blob/dev/CHANGELOG.rst#090-2014-09-15), so `APACHE-2.0` isn't needed.